lib/buildkite/pipelines/command.rb

```diff
@@ -43,7 +45,8 @@ def initialize(command, subcommand, *args)
       end
 
       def run
-        system(*to_a)
+        stdout, _, _ = Open3.capture3(*to_a)
+        return stdout
```

Comment:
I think the meta_data command is the only one where we'd want the stdout back. The others should continue to be `true`/`false`. I think you can either make another run method (eg `run_and_capture`) or make `run` take in an argument (eg `run(capture: false)`) and use it for meta data.

Reply:
If capture, return stdout. If not capture, you should return whether or not the process exit was successful.